holoviews/plotting/renderer.py

```diff
@@ -183,6 +183,14 @@ def get_plot(self_or_cls, obj, renderer=None):
         """
         Given a HoloViews Viewable return a corresponding plot instance.
         """
+        if isinstance(obj, DynamicMap) and obj.unbounded:
+            dims = ', '.join('%r' % dim for dim in obj.unbounded)
+            msg = ('DynamicMap cannot be displayed without explicit indexing '
+                   'as {dims} dimension(s) are unbounded. '
+                   '\nSet dimensions bounds with the DynamicMap redim.range '
+                   'or redim.values methods.')
+            raise SkipRendering(msg.format(dims=dims))
```

Comment:
I guess bokeh server will have the ``SkipRendering`` exception raised in this case which it has no special handling for. It doesn't have a way to actually 'skip' rendering afaik but this is just a minor issue/comment and it is fine to leave it as is.